### ISSUE TITLE
FBI Injection has busted :/

### ISSUE THREAD

**Issue by lavanoid** [roxas75/rxTools#159]
I went to install FBI into H&S but it freezes on this:

Choose:
    B - check TMD only
    Y - Inject FBI
Opening SecureInfo_A...
Completed!
Region: Europe

It does nothing after that :/

emuNAND 9.2.0-20E, sysNAND is the same.

**Comment by dukesrg** [comment]
@lavanoid I'll hava a look at this tonight (at last).

**Comment by lavanoid** [comment]
Great! :D Oh, I've also seen reports of Smash Bros and MH not working on the latest builds. Thought I'd bring that up instead of making another issue (I report so much stuff, so it seems :P)

**Comment by dukesrg** [comment]
no, ssb/mh4 relies on reboot patch

**Comment by lavanoid** [comment]
Ah. Welp, just passing on what other's have said :)

**Comment by dukesrg** [comment]
@lavanoid so far only found two missed ConsoleShow which prevents progress report. Did you previously used that feature and trying this time waited the same amount of time to complete?

**Comment by lavanoid** [comment]
I haven't tested the latest commit, if that's what you're asking. I haven't tried it since I reported this issue. Back when it was busted, I has waited for at least 2 minutes but it still didn't progress.

At this moment in time, I restored my 9.9 sysNAND backup so unable to test. Sill test once I restore 9.2

**Comment by dukesrg** [comment]
@lavanoid Might be fixed with #174

**Comment by lavanoid** [comment]
@dukesrg Will have to see if anyone has issues with it as I currently can't test :/

**Comment by lavanoid** [comment]
Hangs on "Backing up Health & Safety info..." Nothing else after that :/

**Comment by lavanoid** [comment]
Built the latest still hasn't fixed it :/ Freezes on that:

https://dl.dropbox.com/s/p0hkfewzypb6krf/IMAG0161.jpg?dl=0

Sorta weird how it says "Injecting to " but doesn't say if it's using sys or emunand.

**Comment by Gadorach** [comment]
There's still work being done to translate the app right now, don't expect it to work right just yet. Use B9 or earlier for proper injection. Once the translation work is done, the app will be fixed.

**Comment by lavanoid** [comment]
Okey doke :) Thought I'd help debug the problem by showing my reports, anyways :D 

Can't wait to see how this project is gonna turn out :)

**Comment by dukesrg** [comment]
One more fix in FBI inject added today

**Comment by lavanoid** [comment]
Yup, FBI seems to inject now :) It spazzes out after injecting though. After injecting, it goes like this:

https://dl.dropbox.com/s/6o8jqrdcrtwsu8p/IMG_20150814_053838.jpg?dl=0

Then when I press "B", it asks to press "A", so I do that and it returns to the "Advanced" menu. Cute.

**Comment by kot2002** [comment]
#239 fixes this.

**Comment by lavanoid** [comment]
Okey doke :D
